central/cluster/datastore/datastore_impl.go

```diff
@@ -753,7 +753,7 @@ func (ds *datastoreImpl) getAlerts(ctx context.Context, deploymentID string) ([]
 	q := pkgSearch.NewQueryBuilder().
```

Comment:
Just pointing out that this is how we should have required the queries to be built when first writing this stuff so we don't have to do this.